codeflow.js

```diff
@@ -1,18 +1,80 @@
-var discussionHeaders = document.getElementsByClassName("gh-header-number");
-var discussionHeader = discussionHeaders[0];
-var prlink = document.location;
-
-var codeflowElement = document.createElement("span");
+(function(){

```

Comment:
generally looks great. I made some changes yesterday and today to optimize the link for various scenarios such as # to specific changes and files tab/files change links. Thus this needs a minor update to ensure those changes stick.

Reply:
I've re-applied the changes that you made into the new code I wrote.